CHANGELOG.md

```diff
@@ -102,6 +102,17 @@ If you're importing JavaScript modules individually, you should check any refere
 
 This change was introduced in [pull request #1836: Rename exported JavaScript modules to include component name](https://github.com/alphagov/govuk-frontend/issues/1836)].
 
+#### Update the HTML for hints
+
+In v3.8.0, we updated the HTML for hints to use a <div> rather than a <span> element, to allow block elements to be used.
```

Comment:
```suggestion
In [GOV.UK Frontend v3.8.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.8.0), we updated the HTML for hints to use a `<div>` rather than a `<span>` element, to allow the use of block elements.
```
In the v3.8.0 release notes, the link for this change is in the 'Fixes' section, which might make it easy to miss. Wonder is there a case for linking to [#1855](https://github.com/alphagov/govuk-frontend/pull/1855) from the text "updated the HTML for hints...use of block elements"?